ddlambda.go

```diff
@@ -228,6 +231,9 @@ func (cfg *Config) toMetricsConfig() metrics.Config {
 	if !mc.EnhancedMetrics {
 		mc.EnhancedMetrics = strings.EqualFold(enhancedMetrics, "true")
 	}
+	if mc.HttpClientTimeout <= 0 {
+		mc.HttpClientTimeout = time.Second * 5
```

Comment:
Is this needed? If you follow the pattern of `BatchInterval`, I think only need to set the default for `HttpClientTimeout` in one place (`internal/metrics/listener.go`)?